src/components/SearchAndFilter/SearchAndFilter.js

```diff
@@ -1,9 +1,7 @@
-import { CSSTransition } from "react-transition-group";
```

Comment:
It it used elsewhere? Maybe it can be removed from dependencies?

Reply:
I assumed it was as the justification was including it in the first place was to have consistency.. but apparently not. Removing.

Reply:
Sorry I might not have been very clear. This was only added to support the transitions in SearchAndFilter (it needed a way to only run the animations on first render inside the portal).